php/example.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+require 'vendor/autoload.php';
+
+use Ramsey\Uuid\Uuid;
+
+$client = new GuzzleHttp\Client();
+
+# Credentials
+$apikey = $_SERVER["APIKEY"];
+$secret = $_SERVER["SECRET"];
```

Comment:
taka taktyka produkcyjnie jest średnia, ale na potrzeby przykładu do przeżycia

Reply:
ok zostawmy, to tylko przyklad.